stream-chat-android-test/src/main/java/io/getstream/chat/android/test/TestLoggingHelper.kt

```diff
@@ -16,15 +16,31 @@
 
 package io.getstream.chat.android.test
 
+import io.getstream.log.KotlinStreamLogger
+import io.getstream.log.Priority
 import io.getstream.log.StreamLog
-import io.getstream.log.kotlin.KotlinStreamLogger
+import kotlinx.datetime.Clock
+import kotlinx.datetime.LocalDateTime
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.toLocalDateTime
 
 public object TestLoggingHelper {
 
-    public fun initialize(now: () -> Long = { System.currentTimeMillis() }) {
+    public fun initialize() {
         StreamLog.setValidator { _, _ -> true }
-        StreamLog.install(
-            KotlinStreamLogger(now = now),
-        )
+        StreamLog.install(StreamTestLogger())
     }
 }
+
+internal class StreamTestLogger : KotlinStreamLogger() {
+
+    override fun log(priority: Priority, tag: String, message: String, throwable: Throwable?) {
+        if (throwable != null) {
+        }
```

Comment:
Can we remove this `if` block?
To avoid `detekt` warning, you can add something like `// No-Op` to keep the method as empty without any warning

Reply:
Updated :)